docs/codeql/codeql-overview/codeql-changelog/codeql-cli-2.10.0.rst

```diff
@@ -14,7 +14,7 @@ This is an overview of changes in the CodeQL CLI and relevant CodeQL query and l
 Security Coverage
 -----------------
 
-CodeQL 2.10.0 runs a total of 339 security queries when configured with the Default suite (covering 142 CWE). The Extended suite enables an additional 104 queries (covering 30 more CWE). 4 security queries have been added with this release.
+CodeQL 2.10.0 runs a total of 339 security queries when configured with the Default suite (covering 142 CWE). The Extended suite enables an additional 104 queries (covering 30 more CWE). A list of queries for each suite and language `is available here <https://docs.github.com/en/code-security/code-scanning/managing-your-code-scanning-configuration/codeql-query-suites#queries-included-in-the-default-and-security-extended-query-suites>`__. 4 security queries have been added with this release.
```

Comment:
I don't think we should be adding this link to old change logs. We certainly don't list the queries available for CodeQL 2.10.0 anywhere.

It might make more sense to add this only to the doc for the current release, and always to remove it from the previous release?

Reply:
Can we move it to the index page instead?

Reply:
That sounds like a much better idea 👍🏻

Reply:
> That sounds like a much better idea 👍🏻